src/main/java/john/ui/MainWindow.java

```diff
@@ -0,0 +1,63 @@
+package john.ui;
+
+import java.util.Objects;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import john.John;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private John john;
+
+    private Image userImage = new Image(Objects.requireNonNull(
+            this.getClass().getResourceAsStream("/images/user.png")));
+    private Image johnImage = new Image(Objects.requireNonNull(
+            this.getClass().getResourceAsStream("/images/john.png")));
```

Comment:
You can consider setting this to final since it is unlikely the name should change